Code/Engine/Foundation/Types/TypeTraits.h

```diff
@@ -193,6 +193,8 @@ EZ_DEFINE_AS_POD_TYPE(ezUInt8);
 EZ_DEFINE_AS_POD_TYPE(ezUInt16);
 EZ_DEFINE_AS_POD_TYPE(ezUInt32);
 EZ_DEFINE_AS_POD_TYPE(ezUInt64);
+EZ_DEFINE_AS_POD_TYPE(long);
```

Comment:
long and unsigned long are already defined below

Reply:
Removed the duplicate commit, had been integrated here before.